spec/project.schema.json

```diff
@@ -6,20 +6,25 @@
       "type": "string"
     },
     "samples": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "hash": {
-            "type": "string"
+      "type": "object",
+      "patternProperties": {
+        "^[a-zA-Z0-9]*$": {
+          "type": "object",
+          "properties": {
+            "hash": {
+              "type": "string"
+            },
+            "cluster": {
+              "type": ["integer", "string"]
+            },
+            "sketch": { "$ref": "sketch.schema.json" },
+            "failedReasons": {
+              "type": "string"
```

Comment:
Might be nicer to return this as an array rather than making the client do the split?